docs/Makefile

```diff
@@ -18,9 +18,3 @@ help:
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
 	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
-
-clean:
```

Comment:
Why do we not want to keep the clean target?

Reply:
The sphinx-quickstart installed this. The previous version of `Makefile` was probably also installed by sphinx. I had deleted the previous version, starting with a new empty `docs` directory, so this is generated by sphinx.